CRM/Mailing/MailStore.php

```diff
@@ -69,12 +69,8 @@ public static function getStore($name = NULL) {
   private static function getProtocolDefaults($protocol) {
     switch ($protocol) {
       case 'IMAP':
-      case 'IMAP_XOAUTH2':
         return [
-          // For backward compat with pre-release XOAuth2 configurations
-          'auth' => $protocol === 'IMAP_XOAUTH2' ? 'XOAuth2' : 'Password',
-          // In a simpler world:
-          // 'auth' => 'Password',
+          'auth' => 'Password',
```

Comment:
@demeritcowboy won't this break the XOUATH2 integration?  or am I confused by things here?

Reply:
See it is confusing! No because IMAP_XOAUTH2 is never used. Whether to use oauth2 with imap comes from the settings stored in a different place, via `$mailSettings['auth']` below.

I also tested this with the same setup I had used for testing oauth2 before. Still seems to work.